roles/kubernetes-apps/ansible/templates/netchecker-agent-ds.yml.j2

```diff
@@ -12,6 +12,9 @@ spec:
       labels:
         app: netchecker-agent
     spec:
+{% if pod_priority_assignment|default(true) and kube_version|version_compare('v1.11.2', '>=') and netcheck_namespace == 'kube-system'  %}
```

Comment:
I'm not following the namespace conditionals. Why do we need that?

Reply:
Priority class system-node-critical & system-cluster-critical are only defined for kube-system namespaces.

Reply:
A PriorityClass is not namespaced. They should be available cluster-wide.